api-reference/_hidden/GridBaseColumn/name.md

```diff
@@ -5,7 +5,8 @@ default: undefined
 ---
 ---
 ##### shortDescription
-Specifies the identifier of the column.
+Specifies a unique identifier of the column. The value is inherited from the [dataField]({basewidgetpath}/Configuration/columns/#dataField) by default.
```

Comment:
```suggestion
Specifies the column's unique identifier. This value is inherited from the [dataField]({basewidgetpath}/Configuration/columns/#dataField).
```

Reply:
This value is inherited from the [dataField] _only if the name option is not set in code_. Thus can the sentence be "Specifies the column's unique identifier. If not set in code, this value is inherited from the [dataField]()"?

Reply:
I agree with your suggestion, but your original version "The value is inherited from the [dataField]({basewidgetpath}/Configuration/columns/#dataField) by default" and "If not set in code, this value is inherited from the [dataField]({basewidgetpath}/Configuration/columns/#dataField)" is different.